### PR TITLE
collectd doc

### PR DESCRIPTION
**Pull request by paha**

### CODE REVIEW

**Comment by jgruica** [comment]
@paha https://github.com/replicatedhq/kurl.sh/blob/master/README.md#updating-documentation-flags
this is a patter when updating documentation

File: src/markdown-pages/addons/collectd.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+path: "/docs/add-ons/collectd"
+date: "2020-11-02"
+linktitle: "Collectd addon Add-On"
+weight: 31
+title: "Collectd Add-On"
+addOn: "collectd"
+---
+[collectd](https://collectd.org/) is deployed on kurl host gathering system statistics to track system health and find performance bottlenecks.
```
Comment (areed) [inline src/markdown-pages/addons/collectd.md:9]:
```suggestion
[collectd](https://collectd.org/) gathers system statistics on kURL hosts to track system health and find performance bottlenecks.
```

**Review by areed — APPROVED** [review]

File: static/versionDetails.json [hunk 2]
```diff
@@ -456,5 +456,26 @@
...
     }
+  ],
+  "collectd": [
+    {
+      "flag": "version",
+      "description": "The version of collectd to be installed",
+      "type": "string"
+    }
+  ],
+  "certManager": [
```
Comment (jgruica) [inline static/versionDetails.json:467]: can we make this as "cert-manager" to match what it is in yaml?

File: static/versionDetails.json [hunk 3]
```diff
@@ -456,5 +456,26 @@
...
+    {
+      "flag": "version",
+      "description": "The version of Certificate Manager to be installed",
+      "type": "string"
+    }
+  ],
+  "metricsServer": [
```
Comment (jgruica) [inline static/versionDetails.json:474]: same here: "metrics-server"

**Review by jgruica — CHANGES_REQUESTED** [review]

File: src/markdown-pages/addons/collectd.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+path: "/docs/add-ons/collectd"
+date: "2020-11-02"
+linktitle: "Collectd addon Add-On"
```
Comment (jgruica) [inline src/markdown-pages/addons/collectd.md:4]:
```suggestion
linktitle: "Collectd Add-On"
```

**Review by jgruica — CHANGES_REQUESTED** [review]

File: src/markdown-pages/addons/collectd.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+---
+[collectd](https://collectd.org/) gathers system statistics on kURL hosts to track system health and find performance bottlenecks.
+
+## Advanced Install Options
+
+```yaml
+spec:
+  collectd:
+    version: 0.0.1
+```
```
Comment (jgruica) [inline src/markdown-pages/addons/collectd.md:17]: flags-table

**Review by jgruica — CHANGES_REQUESTED** [review]

**Review by jgruica — APPROVED** [review]

**Pull request merged**